var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -31,13 +31,11 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
 
     version("master", branch="master", submodules=True)
     version(
-        "5.12.0-RC3", sha256="6aaa46ff295126707294482e6ba24bd0ec0d68cf6bb5f56f145f8bcc53fc3f70"
-    )
-    version(
-        "5.11.2",
-        sha256="5c5d2f922f30d91feefc43b4a729015dbb1459f54c938896c123d2ac289c7a1e",
+        "5.12.0",
+        sha256="d289afe7b48533e2ca4a39a3b48d3874bfe67cf7f37fdd2131271c57e64de20d",
         preferred=True,
```

Comment:
If the preferred version is simply the latest, we don't need to specify it as preferred since that's the default behavior.

Reply:
It is there for two reasons. We add RC releases we don't want it to auto select the newer version, having that all the time hurts nothing and makes it easier to add RCs quickly for testing without accidentally changing the stable default. There is also a master version, which I think gets selected by default as the newest version, which we do not want.

Reply:
Infinite versions (`master`, `main`, `develop`) are not chosen as preferred versions by default.